pages/content/amp-dev/documentation/guides-and-tutorials/start/bento_guide.md

```diff
@@ -174,120 +179,140 @@ The example below triggers [`amp-accordion`'s toggle action](../../../documentat
-    <img width="150px" height="100px" src="https://amp.dev/static/samples/img/product2_1024x682.jpg" />
+    <img
+      width="150px"
+      height="100px"
+      src="https://amp.dev/static/samples/img/product2_1024x682.jpg"
+    />
   </section>
 </amp-accordion>
-<button id="toggle">
-  Toggle Accordion
-</button>
+<button id="toggle">Toggle Accordion</button>
 <script>
   const toggleAccordion = async () => {
     const accordion = document.querySelector('#my-accordion');
     const api = await accordion.getApi();
     api.toggle();
-  }
+  };
   const toggleButton = document.querySelector('#toggle');
   toggleButton.addEventListener('click', toggleAccordion);
 </script>
 ```
-[/example]
 
-Read each [component's reference documentation](#available-bento-components) for a full list of available actions and events. 
+[/example]
 
+Read each [component's reference documentation](#available-bento-components) for a full list of available actions and events.
 
-# Available Bento components  <a name="available-bento-components"></a>
+# Available Bento components <a name="available-bento-components"></a>
 
 Bento supported AMP components are listed below:
 
-*   [amp-accordion](../../../documentation/components/reference/amp-accordion-v1.0.md)
-*   [amp-base-carousel](../../../documentation/components/reference/amp-base-carousel-v1.0.md)
-*   [amp-inline-gallery](../../../documentation/components/reference/amp-inline-gallery-v1.0.md)
-    *   amp-inline-gallery-pagination
-    *   amp-inline-gallery-thumbnails
-*   [amp-stream-gallery](../../../documentation/components/reference/amp-stream-gallery-v1.0.md)
-*   [amp-date-countdown](../../../documentation/components/reference/amp-date-countdown-v1.0.md)
-*   [amp-date-display](../../../documentation/components/reference/amp-date-display-v1.0.md)
-*   [amp-fit-text](../../../documentation/components/reference/amp-fit-text-v1.0.md)
-*   [amp-instagram](../../../documentation/components/reference/amp-instagram-v1.0.md)
-*   [amp-lightbox](../../../documentation/components/reference/amp-lightbox-v1.0.md)
-*   [amp-selector](../../../documentation/components/reference/amp-selector-v1.0.md)
-*   [amp-social-share](../../../documentation/components/reference/amp-social-share-v1.0.md)
-*   [amp-timeago](../../../documentation/components/reference/amp-timeago-v1.0.md)
-*   Video components
-    *   [amp-video](../../../documentation/components/reference/amp-video-v1.0.md)
-    *   [amp-youtube](../../../documentation/components/reference/amp-youtube-v1.0.md)
+- [amp-accordion](../../../documentation/components/reference/amp-accordion-v1.0.md)
+- [amp-base-carousel](../../../documentation/components/reference/amp-base-carousel-v1.0.md)
+- [amp-inline-gallery](../../../documentation/components/reference/amp-inline-gallery-v1.0.md)
+  - amp-inline-gallery-pagination
+  - amp-inline-gallery-thumbnails
+- [amp-stream-gallery](../../../documentation/components/reference/amp-stream-gallery-v1.0.md)
```

Comment:
I can never decide with these two - should they stay with `amp-base-carousel` since they are all carousels, or should they be split up so the list can be alphabetized?

Reply:
We could auto generate this list and then they would be split up anyway.